README.md

```diff
@@ -168,12 +170,21 @@ Default: `true`
 
 If `false`, no caches will be used.
 
+##### sync
+
+Type: `boolean`
+Default: `false`
+
+If `true`, config will be loaded synchronously.
+
 ##### transform
 
-Type: `Function` returning a Promise
+Type: `Function` returning a Promise(`sync` is false)
 
 A function that transforms the parsed configuration. Receives the result object with `config` and `filepath` properties, and must return a Promise that resolves with the transformed result.
 
+If the `sync` option is passed as `true`, transform need not be a function which returns a promise.
```

Comment:
You are right. While it wouldn't break anything, having a promise returning function for `transform` would render the `sync` option moot.